library/src/main/res/layout/chucker_transaction_item_image.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ImageView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/chuckerTransactionResponseBinaryData"
+    android:id="@+id/binaryData"
```

Comment:
Thanks for driving this forward @vbuberen @MiSikora 
Out of curiosity, why we renamed back all the IDs to don't have `chucker` in the name? Was this because we wanted to have a cleaner code (due to view binding)?

Reply:
When we used `findViewById` we could get into trouble when user has the same id for some view as Chucker. In such cases Chucker would crash, since, for example, instead of `toolbar` from `activity_transaction` it would find a `toolbar` from some layout in user's project. Due to this issue I renamed all ids for views for these long `chucker.....` items.

Now, when we switched to `ViewBinding` we can be 100% sure that the generated binding class has references to proper views, so we can change their ids to more concise ones.

This was one of the reasons I decided to switch Chucker to `ViewBinding`.

Reply:
Awesome, thanks for the clarification